src/replication.c

```diff
@@ -828,6 +828,7 @@ struct appendFollower
     raft_index index;  /* Index of the first entry in the request. */
     struct raft_append_entries args;
     struct raft_io_append req;
+    raft_term term;    /* Used to check for state changes. */
```

Comment:
The `struct raft_io_append` above already contains the term of the request, so this is not needed I think.

Reply:
sorry, I meant the `struct raft_append_entries`